src/rust/plugin-registry/tests/test_create_plugin.rs

```diff
@@ -21,6 +21,7 @@ async fn test_create_plugin() -> Result<(), Box<dyn std::error::Error>> {
     let tenant_id = uuid::Uuid::new_v4();
```

Comment:
This file shouldn't be in this PR, since it isn't related to organization management.